packages/dapp/components/staking/bonding-shares-explorer.tsx

```diff
@@ -44,60 +44,63 @@ type Actions = {
 const USD_TO_LP = 0.7460387929;
 const LP_TO_USD = 1 / USD_TO_LP;
 
-export const BondingSharesExplorerContainer = ({ managedContracts, web3Provider, walletAddress, signer }: LoadedContext) => {
+export const BondingSharesExplorerContainer = ({ protocolContracts, web3Provider, walletAddress, signer }: LoadedContext) => {
   const [model, setModel] = useState<Model | null>(null);
   const [, doTransaction] = useTransactionLogger();
   const [, refreshBalances] = useBalances();
-  // cspell: disable-next-line
-  const { staking: bonding, masterChef, stakingToken: bondingToken, dollarMetapool: metaPool } = managedContracts;
 
   useAsyncInit(fetchSharesInformation);
   async function fetchSharesInformation(processedShareId?: ShareData["id"]) {
-    console.time("BondingShareExplorerContainer contract loading");
-    const currentBlock = await web3Provider.getBlockNumber();
-    // cspell: disable-next-line
-    const blockCountInAWeek = +(await bonding.blockCountInAWeek()).toString();
-    const totalShares = await masterChef.totalShares();
     // cspell: disable-next-line
-    const bondingShareIds = await bondingToken.holderTokens(walletAddress);
-    const walletLpBalance = await metaPool.balanceOf(walletAddress);
-
-    const shares: ShareData[] = [];
-    await Promise.all(
+    const { stakingFacet, chefFacet, stakingShare: bondingToken, curveMetaPoolDollarTriPoolLp } = await protocolContracts;
+    if (stakingFacet) {
+      console.time("BondingShareExplorerContainer contract loading");
+      const currentBlock = await web3Provider.getBlockNumber();
       // cspell: disable-next-line
-      bondingShareIds.map(async (id: BigNumber) => {
+      const blockCountInAWeek = +(await stakingFacet.blockCountInAWeek()).toString();
+      const totalShares = await chefFacet?.totalShares();
+      // cspell: disable-next-line
+      const bondingShareIds = await bondingToken?.holderTokens(walletAddress);
+
+      const walletLpBalance = await curveMetaPoolDollarTriPoolLp?.balanceOf(walletAddress);
+
+      const shares: ShareData[] = [];
+      await Promise.all(
         // cspell: disable-next-line
-        const [ugov, bond, bondingShareInfo, tokenBalance] = await Promise.all([
-          // cspell: disable-next-line
-          masterChef.pendingGovernance(id),
+        bondingShareIds.map(async (id: BigNumber) => {
           // cspell: disable-next-line
-          bondingToken.getStake(id),
-          masterChef.getStakingShareInfo(id),
-          // cspell: disable-next-line
-          bondingToken.balanceOf(walletAddress, id),
-        ]);
+          const [ugov, bond, bondingShareInfo, tokenBalance] = await Promise.all([
```

Comment:
We don't use token symbols. Instead change them to what they stand for e.g. `governanceToken`

https://github.com/ubiquity/ubiquity-dollar/blob/development/.github/CONTRIBUTING.md